proposals/XXXX-Adding-new-parameters-for-transport-switching.md

```diff
@@ -0,0 +1,122 @@
+# Adding new parameters for transport switching
+
+* Proposal: [SDL-XXXX](XXXX-Adding-new-parameters-for-transport-switching.md)
+* Author: [Kazuki Sugimoto(Nexty)](https://github.com/Kazuki-Sugimoto)
+* Status: **Awaiting Review**
+* Impacted Platforms: [ Core / iOS / Java Suite / RPC / HMI ]
```

Comment:
Should JavaScript Suite be included here as well?  Typically, all app libraries are impacted by RPC changes.

Reply:
Added JavaScript Suite.